ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -282,10 +287,10 @@ public ClusterMapConfig(VerifiableProperties verifiableProperties) {
     clusterMapClusterChangeHandlerType =
         verifiableProperties.getString("clustermap.cluster.change.handler.type", "SimpleClusterChangeHandler");
     clusterMapDcsZkConnectStrings = verifiableProperties.getString("clustermap.dcs.zk.connect.strings", "");
```

Comment:
minor: in future PR, we can replace all config names with string constants (like what you did in this PR to make them consistent)

Reply:
Yes.  I didn't do it in this PR to keep the scope limited.